sympy/stats/drv.py

```diff
@@ -38,10 +38,61 @@ class SingleDiscreteDistribution(DiscreteDistribution, NamedArgsMixin):
 
     Serves as superclass for PoissonDistribution etc....
 
-    Provides methods for pdf, cdf, and sampling
+    Provides methods for pdf, cdf, and sampling.
 
-    See Also:
-        sympy.stats.crv_types.*
+    Explanation
+    ===========
+
+    The distribution object can be called in two ways to evaluate the
+    probability density:
+
+    1. ``distribution(x)``:
+       Returns the probability density at x, with support checking enforced
+       via Piecewise. For symbolic arguments where support membership cannot
+       be determined, this returns a Piecewise expression that evaluates to
+       the pdf formula when x is in the support, and 0 otherwise.
+
+    2. ``distribution.pdf(x)``:
+       Returns the raw probability density formula without support checking.
+       This is useful for symbolic manipulation and integration where the
+       Piecewise wrapper would be cumbersome.
```

Comment:
This should probably refer to probability mass function rather than probability density function. It is confusing because it should not really be called `.pdf` at all.